components/Ripple/src/MDCRippleView.m

```diff
@@ -61,7 +61,12 @@ - (void)commonMDCRippleViewInit {
   self.backgroundColor = [UIColor clearColor];
   self.autoresizingMask = UIViewAutoresizingFlexibleHeight | UIViewAutoresizingFlexibleWidth;
 
-  _rippleColor = [[UIColor alloc] initWithWhite:0 alpha:kRippleDefaultAlpha];
+  static UIColor *defaultRippleColor;
```

Comment:
Do we need this change? It looks like it's going from a single initialized color to a single initialized color? I don't think we expect clients to call the `initializer` multiple times do we? The other two instances, I understand because we recreate layers and then the method was reinitializing but this change I am confused about.

Reply:
This change will result in one color instance being shared across all ripple and ink instances, so there is potential for this adding up. I’d love to see the benefits of this change quantified in the PR description though so that we can measure the impact of it.

Reply:
If that is the case, shouldn't we call `commonMDCRippleViewInit` in the `dispatch_once` instead of just the color?

Reply:
I will revert this change. Since this is neither my code nor my motivation (it came from an internal client), I don't have the energy to do performance analyses at that level.